cypress/e2e/seededFlows/adminFlows/articles/privilegedReactions.spec.js

```diff
@@ -0,0 +1,181 @@
+  });
+
+  it('should not contain quality reactions', () => {
+    cy.findByRole('link', { name: 'Quality reactions' }).click();
+    cy.findByText('Article has no quality reactions by trusted users.').should(
+      'exist',
+    );
+  });
+
+  it('should display the correct thumb up count', () => {
+    cy.get('.flex .crayons-card:nth-child(1) .fs-s').should('contain', '0');
+  });
+
+  it('should display the correct thumb down count', () => {
+    cy.get('.flex .crayons-card:nth-child(2) .fs-s').should('contain', '0');
+  });
+
+  it('should display the correct vomit count', () => {
+    cy.get('.flex .crayons-card:nth-child(3) .fs-s').should('contain', '0');
+  });
+
+  it('should display the correct score', () => {
+    cy.get('.flex .crayons-card:nth-child(4) .fs-s').should('contain', '0');
+  });
+});
+
+describe('Article flagged by a user', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.intercept('POST', '/admin/reactions/*', (req) => {
+      req.reply((res) => {
+        const { id } = req.body;
+        const response = {
+          statusCode: 200,
+          body: {
+            outcome: 'Success',
+          },
+        };
+
+        if (req.body.removeElement === true) {
+          cy.get(`#${id}`).then(($element) => {
+            $element.remove();
+          });
+          cy.get(`#js__reaction__div__hr__${id}`).then(($element) => {
+            $element.remove();
+          });
+        } else {
+          cy.reload();
+        }
+
+        res.send(response);
+      });
+    });
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/admin/content_manager/articles/1'); // This article contains one flagged reaction.
+      });
+    });
+  });
+
+  it('should display the correct default data', () => {
+    cy.get('.flex .crayons-card:nth-child(3) .fs-s').should('contain', '1');
+  });
+
+  it('should display the correct user name', () => {
+    cy.get('.flex .crayons-subtitle-3').should(
+      'contain',
+      'Trusted User 1 \\:/',
+    );
+  });
+
+  it('should display the open flag status', () => {
+    cy.get('.flex .c-indicator').should('be.visible').as('flagStatus');
+    cy.get('@flagStatus')
+      .should('have.class', 'c-indicator--warning')
+      .and('contain', 'Open');
+  });
+
+  it('should display the correct score for open flag', () => {
+    cy.get('.flex .crayons-card:nth-child(2) .fs-s').should('contain', '-50');
+  });
+
+  it('should open the dropdown on button click', () => {
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('be.visible');
+  });
+
+  it('should open the dropdown on button click and close the dropdown on same button click', () => {
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('be.visible');
+
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('not.visible');
+  });
```

Comment:
E2E is way slower than unit test, so it's more acceptable to test more than one aspect per block. In this case, your `should open the dropdown on button click and close the dropdown on same button click` is already also testing what `should open the dropdown on button click` is doing, making the line 125 redundant.

Reply:
Updated the test to write it like this:
```
  it('should close the dropdown on same button click twice', () => {
    cy.get('.c-btn--icon-alone').click();
    cy.get('.c-btn--icon-alone').click();
    cy.get('.crayons-dropdown').should('not.visible');
  });
```